### PR TITLE
conjure-up 2.2-beta2, 2.1 updates

### PR DESCRIPTION
**Pull request by adam-stokes**
Update devel to latest 2.2-beta2

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by battlemidget** [comment]
Not entirely sure whats going on with the build here

**Comment by dunn** [comment]
SHA mismatch:
```
==> Downloading from https://codeload.github.com/conjure-up/conjure-up/tar.gz/2.1
Warning: Formula reports different SHA256: 933f94c21b999fd24b4c304ba27c59bde9c83f478f5ee6a45cb8f265aaac2658
Downloaded to: /Users/brew/Library/Caches/Homebrew/conjure-up-2.1.tar.gz
SHA256: 4f60e8cbbb7626c55b63928c65255cddad7df720f04ddb87f9888e1c3de1a44d

File: Formula/conjure-up.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class ConjureUp < Formula
   desc "Big software deployments so easy it's almost magical."
   homepage "http://conjure-up.io"
   url "https://github.com/conjure-up/conjure-up/archive/2.1.tar.gz"
-  sha256 "933f94c21b999fd24b4c304ba27c59bde9c83f478f5ee6a45cb8f265aaac2658"
+  sha256 "4f60e8cbbb7626c55b63928c65255cddad7df720f04ddb87f9888e1c3de1a44d"
```
Comment (dunn) [inline Formula/conjure-up.rb]: Oh, I didn't realize the mismatch was for the stable.  Do you know why it changed?

Comment (battlemidget) [inline Formula/conjure-up.rb]: We had some critical bug fixes that needed to be put in to the 2.1 branch.

Comment (dunn) [inline Formula/conjure-up.rb]: Gotcha.  In such cases it'd be better (for us at least) to have a 2.1.1 release instead of force-pushing.

Comment (dunn) [inline Formula/conjure-up.rb]: Could you add `revision 1` underneath this line, so that the updated version is pushed out to everyone?

Comment (battlemidget) [inline Formula/conjure-up.rb]: Thanks, our 2.2 GA is coming April 20th and *hopefully* that is the next major update to the stable branch

**Comment by battlemidget** [comment]
@fxcoudert Looks like the update you made to this PR caused the build to fail?

**Comment by fxcoudert** [comment]
Merged, thanks for the PR, in the future please avoid releasing a new source tarball under the same name as this create conflicts for us and trust issues for users.

**Comment by battlemidget** [comment]
Thanks, I'll make sure to bump patch level if required next time

**Pull request closed without merging**